Main.java

```diff
@@ -9,7 +10,13 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // TODO code application logic here
+    Scanner input = new Scanner(System.in);
+    
+
+    System.out.println("Please enter the temperature in Celceus");
+   int currentTemperature = input.nextInt();
+   int farenheight = currentTemperature*9/5+32;
+   System.out.println("The current temperature is " + farenheight + ".");

```

Comment:
While this works, you are missing comments in your code. Be sure to comment all of the "steps" needed in your program. See any of the examples in the lesson for what it should look like. Also, the reason you have not passed any of the test cases is because your formatting does not match the sample. Make sure you are trying to replicate the example as closely as possible.